### PR TITLE
core(audits): point perf audit description links to web.dev

### PR DESCRIPTION
**Pull request by mfriesenhahn**
**Summary**
Points "Learn more" links in performance audits to relevant web.dev guides.

**Note**
web.dev currently doesn't have audit-oriented posts for the **Keep request counts low and transfer sizes small** and **Third-party Usage** audits. However, there are posts in the Learn section whose content could be repurposed. We're working to figure out how quickly that can be done.

### CODE REVIEW

File: lighthouse-core/audits/metrics/first-cpu-idle.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const UIStrings = {
   /** Description of the First CPU Idle metric, which marks the time at which the page has displayed content and the CPU is not busy executing the page's scripts. This is displayed within a tooltip when the user hovers on the metric name to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'First CPU Idle marks the first time at which the page\'s main thread is ' +
     'quiet enough to handle input. ' +
```
Comment (mfriesenhahn) [inline lighthouse-core/audits/metrics/first-cpu-idle.js]:
```suggestion
    'quiet enough to handle input.  [Learn more](https://web.dev/first-cpu-idle).',
```

File: lighthouse-core/audits/metrics/first-cpu-idle.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const UIStrings = {
...
-    '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/first-interactive).',
+    '[Learn more](https://web.dev/first-cpu-idle).',
```
Comment (mfriesenhahn) [inline lighthouse-core/audits/metrics/first-cpu-idle.js]:
```suggestion
```

**Comment by mfriesenhahn** [comment]
@patrickhulce, I've copied your comments to (web.dev issue 1344](https://github.com/GoogleChrome/web.dev/issues/1344) for tracking.

**Comment by paulirish** [comment]
i resolved the merge conflicts. we'll merge this PR whenever it's green.

naturally, i would like to see patrick's feedback addressed within the articles. :)

**Comment by mfriesenhahn** [comment]
> naturally, i would like to see patrick's feedback addressed within the articles. :)

Patrick's feedback has been included in the web.dev issue ([#1344](https://github.com/GoogleChrome/web.dev/issues/1344)) we're using to track the doc updates.

**Pull request merged**